csp_billing_adapter/bill_utils.py

```diff
@@ -333,6 +349,20 @@ def filter_usage_records_in_billing_period(
     return filtered_records
 
 
+def get_errors(status: dict):
+    """
+    Return a list of all error messages
+
+    Parse metering status dictionary for all error messages
+    and return a list. If none return an empty list.
+    """
+    errors = []
+
+    for dimension, data in status.items():
+        if 'error' in data:
+            errors.append(data['error'])
+
```

Comment:
I think you are missing a return statement here that returns the collected errors.

Reply:
```suggestion

    return errors

```